examples/kubernetes-kubenet-containerd.json

```diff
@@ -2,7 +2,6 @@
 	"apiVersion": "vlabs",
 	"properties": {
 		"orchestratorProfile": {
-			"orchestratorType": "Kubernetes",
```

Comment:
Nit: this file could be JSON pretty-printed like the others to use two spaces.

Reply:
Agree, there's a bunch of different JSON here.

Probably will advocate that we consider getting rid of these examples files and instead add more documentation that refers to the `test_cluster_config` directory, so folks know what cluster configs are regularly being tested.